modules/kafka/kafka-dest-driver.h

```diff
@@ -35,27 +35,33 @@ typedef struct
   LogTemplateOptions template_options;
   LogTemplate *key;
   LogTemplate *message;
+  LogTemplate *topic_name;
+  GHashTable *topic_hash_map;
+  GMutex lock;
```

Comment:
This could be `topics_lock` or something like that, to better show its intention.